intern/remote/cv/remote.py

```diff
@@ -0,0 +1,278 @@
+            layer_type: (str) typically "image" or "segmentation"
+            data_type: (str) e.g. "uint8", "uint16", "uint32", "float32"
+            resolution: int (x,y,z), x,y,z voxel dimensions in nanometers
+
+            volume_size: int (x,y,z), extent of dataset in cartesian space from voxel_offset
+
+        Optional:
+            voxel_offset: int (x,y,z), beginning of dataset in positive cartesian space
+            encoding: (str) "raw" for binaries like numpy arrays, "jpeg", "png"
+            mesh: (str) name of mesh directory, typically "mesh"
+            skeletons: (str) name of skeletons directory, typically "skeletons"
+            chunk_size: int (x,y,z), dimensions of each downloadable 3D image chunk in voxels
+            compressed_segmentation_block_size: (x,y,z) dimensions of each compressed sub-block
+                (only used when encoding is 'compressed_segmentation')
+            max_mip: (int), the maximum mip level id.
+            factor: (tuple), the downsampling factor for each mip level
+
+        Returns: dict representing a single mip level that's JSON encodable
+        """
+        return self._project.create_new_info(
+            num_channels,
+            layer_type,
+            data_type,
+            resolution,
+            volume_size,
+            voxel_offset,
+            encoding,
+            chunk_size,
+            mesh,
+            skeletons,
+            compressed_segmentation_block_size,
+            max_mip,
+            factor,
+        )
+
+    def create_cutout(self, resource, res, x_range, y_range, z_range, data):
+        """
+        Method to upload a cutout of data
+        Args:
+            data (str) : Path to the data
+            vol (CloudVolume) : Existing cloudvolume instance
+            x_range (list) : x range within the 3D space
+            y_range (list) : y range within the 3D space
+            z_range (list) : z range witinn the 3D space
+        Retruns:
+            message (str) : Uploading Data... message
+        """
+        return self._volume.create_cutout(
+            resource, res, x_range, y_range, z_range, data
+        )
+
+    def get_cutout(self, resource, res, x_range, y_range, z_range):
+        """
+        Method to download a cutout of data
+        Args:
+            vol (CloudVolume) : Existing non-empty cloudvolume instance
+            x_range (list) : x range within the 3D space
+            y_range (list) : y range within the 3D space
+            z_range (list) : z range within the 3D space
+        Retruns:
+            data (numpy array) : image stack from the cloud or local system
+        """
+        return self._volume.get_cutout(resource, res, x_range, y_range, z_range)
+
+    def get_info(self, resource):
+        """
+        Returns a JSON of the resource properties.
+
+        Args:
+            resource (CloudVolumeResource object)
+
+        Returns:
+            string: JSON format of properties
```

Comment:
This should be `(str)`. Likewise for methods below.

Reply:
Done